website/src/app/[lang]/[region]/(website)/partners/(components)/PartnerHome.tsx

```diff
@@ -0,0 +1,289 @@
+'use client';
+import { FundraiserBadge, RecipientsBadge } from '@/app/[lang]/[region]/(website)/partners/(components)/PartnerBadges';
+import { CountryBadgeType, RecipientsBadgeType } from '@/app/[lang]/[region]/(website)/partners/(types)/PartnerBadges';
+import {
+	NgoEntryJSON,
+	NgoHomeProps,
+	NgoHoverCardType,
+} from '@/app/[lang]/[region]/(website)/partners/(types)/PartnerCards';
+import { Badge, Separator, Typography } from '@socialincome/ui';
+import { CH, SL } from 'country-flag-icons/react/1x1';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useParams, useRouter } from 'next/navigation';
+import { ReactElement } from 'react';
+
+const country_abbreviations_to_flag_map: Record<string, ReactElement> = {
+	SL: <SL className="h-5 w-5 rounded-full" />,
+	CH: <CH className="h-5 w-5 rounded-full" />,
+};
+function getFlag(abbreviation: string): ReactElement {
+	return country_abbreviations_to_flag_map[abbreviation] || <SL className="h-5 w-5 rounded-full" />;
+}
+export function PartnerHome({
+	ngoArray,
+	partnerSinceTranslation,
+	badgeRecipientTranslation,
+	badgeRecipientTranslationBy,
+	badgeActiveTranslation,
+	badgeFormerTranslation,
+	badgeSuspendedTranslation,
+	fundRaiserTranslation,
+	missionTranslation,
+	foundedTranslation,
+	headquarterTranslation,
+	moreLinksTranslation,
+	websiteTranslation,
+	facebookTranslation,
+	instagramTranslation,
+	linkedinTranslation,
+	youtubeTranslation,
+	permalinkTranslation,
+}: NgoHomeProps) {
+	const router = useRouter();
+	const { orgLongName } = useParams() as { orgLongName: string };
+
+	const deSlugifiedOrgLongName = orgLongName.replaceAll('-', ' ').replace('%26', '&');
```

Comment:
& does appear only at one place...should replaceAll be used here?

Reply:
I guess it makes sense to make it as generic as possible, even though I don’t see many cases for using `&` in one word. However, it could be relevant for other special characters like `.`, for example, in German NGOs, which often have the appendix `e.V.` (like Welthungerhilfe e.V.).